client/data/marketplace/use-marketplace-reviews.ts

```diff
@@ -83,7 +83,7 @@ const fetchMarketplaceReviews = (
 
 const createReview = ( {
 	productType,
-	pluginSlug,
+	slug: productSlug,
```

Comment:
I would just keep `slug` instead of renaming in to `productSlug`.

PS: I think this is the way the text editor handles renaming anyway.

Reply:
>I would just keep slug instead of renaming in to productSlug.
>PS: I think this is the way the text editor handles renaming anyway.

That's right, that was renamed by the editor that way and I didn't realize as it was working fine in the tests. Thanks for the update @WBerredo!